2013/06/19 22:49:23 [error] [php] Creating default object from empty value (C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php:72)
Stack trace:
#0 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): SiteController->runAction()
#1 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): SiteController->runActionWithFilters()
#2 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): SiteController->run()
#3 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#5 C:\xampp\htdocs\questionaire\index.php(13): CWebApplication->run()
REQUEST_URI=/questionaire/index.php?r=site/contact
in C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php (72)
in C:\xampp\htdocs\questionaire\index.php (13)
2013/06/19 22:50:01 [error] [php] Undefined variable: model (C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php:73)
Stack trace:
#0 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): SiteController->runAction()
#1 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): SiteController->runActionWithFilters()
#2 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): SiteController->run()
#3 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#5 C:\xampp\htdocs\questionaire\index.php(13): CWebApplication->run()
REQUEST_URI=/questionaire/index.php?r=site/contact
in C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php (73)
in C:\xampp\htdocs\questionaire\index.php (13)
2013/06/19 22:50:26 [error] [php] Undefined variable: model (C:\xampp\htdocs\questionaire\protected\views\site\contact.php:38)
Stack trace:
#0 C:\xampp\htdocs\yii_main\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii_main\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php(73): SiteController->render()
#3 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): SiteController->actionContact()
#4 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\questionaire\index.php(13): CWebApplication->run()
REQUEST_URI=/questionaire/index.php?r=site/contact
in C:\xampp\htdocs\questionaire\protected\views\site\contact.php (38)
in C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php (73)
in C:\xampp\htdocs\questionaire\index.php (13)
2013/06/19 22:56:06 [error] [php] Undefined variable: view_model (C:\xampp\htdocs\questionaire\protected\views\site\contact.php:38)
Stack trace:
#0 C:\xampp\htdocs\yii_main\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii_main\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php(73): SiteController->render()
#3 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): SiteController->actionContact()
#4 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\questionaire\index.php(13): CWebApplication->run()
REQUEST_URI=/questionaire/index.php?r=site/contact
in C:\xampp\htdocs\questionaire\protected\views\site\contact.php (38)
in C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php (73)
in C:\xampp\htdocs\questionaire\index.php (13)
2013/06/19 22:56:58 [error] [php] Undefined variable: model (C:\xampp\htdocs\questionaire\protected\views\site\contact.php:45)
Stack trace:
#0 C:\xampp\htdocs\yii_main\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii_main\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php(73): SiteController->render()
#3 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): SiteController->actionContact()
#4 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\questionaire\index.php(13): CWebApplication->run()
REQUEST_URI=/questionaire/index.php?r=site/contact
in C:\xampp\htdocs\questionaire\protected\views\site\contact.php (45)
in C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php (73)
in C:\xampp\htdocs\questionaire\index.php (13)
2013/06/19 23:20:47 [error] [exception.CException] exception 'CException' with message 'Property "ContactForm.name" is not defined.' in C:\xampp\htdocs\yii_main\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php(72): CComponent->__set('name', 15)
#1 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): SiteController->actionContact()
#2 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): CController->run('contact')
#6 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController('site/contact')
#7 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\xampp\htdocs\questionaire\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/questionaire/index.php?r=site/contact
---
2013/06/19 23:33:10 [error] [exception.CException] exception 'CException' with message 'Property "ContactForm.names" is not defined.' in C:\xampp\htdocs\yii_main\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii_main\framework\web\helpers\CHtml.php(2255): CComponent->__get('names')
#1 C:\xampp\htdocs\yii_main\framework\web\helpers\CHtml.php(1980): CHtml::resolveValue(Object(ContactForm), 'names')
#2 C:\xampp\htdocs\yii_main\framework\web\helpers\CHtml.php(1257): CHtml::activeInputField('text', Object(ContactForm), 'names', Array)
#3 C:\xampp\htdocs\yii_main\framework\web\widgets\CActiveForm.php(657): CHtml::activeTextField(Object(ContactForm), 'names', Array)
#4 C:\xampp\htdocs\questionaire\protected\views\site\contact.php(42): CActiveForm->textField(Object(ContactForm), 'names')
#5 C:\xampp\htdocs\yii_main\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#6 C:\xampp\htdocs\yii_main\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii_main\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii_main\framework\web\CController.php(782): CController->renderPartial('contact', Array, true)
#9 C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php(73): CController->render('contact', Array)
#10 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): SiteController->actionContact()
#11 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): CController->run('contact')
#15 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController('site/contact')
#16 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#17 C:\xampp\htdocs\questionaire\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/questionaire/index.php?r=site/contact
---
2013/06/19 23:33:36 [error] [exception.CException] exception 'CException' with message 'Property "ContactForm.name" is not defined.' in C:\xampp\htdocs\yii_main\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php(72): CComponent->__set('name', 15)
#1 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): SiteController->actionContact()
#2 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): CController->run('contact')
#6 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController('site/contact')
#7 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\xampp\htdocs\questionaire\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/questionaire/index.php?r=site/contact
---
2013/06/19 23:34:18 [error] [exception.CException] exception 'CException' with message 'Property "ContactForm.name" is not defined.' in C:\xampp\htdocs\yii_main\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php(72): CComponent->__set('name', 15)
#1 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): SiteController->actionContact()
#2 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): CController->run('contact')
#6 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController('site/contact')
#7 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\xampp\htdocs\questionaire\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/questionaire/index.php?r=site/contact
---
2013/06/19 23:34:21 [error] [exception.CException] exception 'CException' with message 'Property "ContactForm.name" is not defined.' in C:\xampp\htdocs\yii_main\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php(72): CComponent->__set('name', 15)
#1 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): SiteController->actionContact()
#2 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): CController->run('contact')
#6 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController('site/contact')
#7 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\xampp\htdocs\questionaire\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/questionaire/index.php?r=site/contact
---
2013/06/19 23:46:49 [error] [exception.CException] exception 'CException' with message 'Property "ContactForm.name" is not defined.' in C:\xampp\htdocs\yii_main\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\questionaire\protected\views\site\contact.php(47): CComponent->__set('name', 'Summer')
#1 C:\xampp\htdocs\yii_main\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#2 C:\xampp\htdocs\yii_main\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii_main\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii_main\framework\web\CController.php(782): CController->renderPartial('contact', Array, true)
#5 C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php(73): CController->render('contact', Array)
#6 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): SiteController->actionContact()
#7 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): CController->run('contact')
#11 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController('site/contact')
#12 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\xampp\htdocs\questionaire\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/questionaire/index.php?r=site/contact
---
2013/06/20 00:05:14 [warning] [application] Failed to set unsafe attribute "names" of "ContactForm".
in C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php (57)
in C:\xampp\htdocs\questionaire\index.php (13)
2013/06/20 00:05:14 [error] [exception.CException] exception 'CException' with message 'Property "ContactForm.name" is not defined.' in C:\xampp\htdocs\yii_main\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii_main\framework\validators\CRequiredValidator.php(51): CComponent->__get('name')
#1 C:\xampp\htdocs\yii_main\framework\validators\CValidator.php(213): CRequiredValidator->validateAttribute(Object(ContactForm), 'name')
#2 C:\xampp\htdocs\yii_main\framework\base\CModel.php(159): CValidator->validate(Object(ContactForm), NULL)
#3 C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php(58): CModel->validate()
#4 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): SiteController->actionContact()
#5 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): CController->run('contact')
#9 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController('site/contact')
#10 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\questionaire\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/questionaire/index.php?r=site/contact
HTTP_REFERER=http://localhost/questionaire/index.php?r=site/contact
---
2013/06/20 23:45:04 [error] [php] include(RWebUser.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii_main\framework\YiiBase.php:421)
Stack trace:
#0 C:\xampp\htdocs\yii_main\framework\YiiBase.php(196): import()
#1 C:\xampp\htdocs\yii_main\framework\base\CModule.php(386): createComponent()
#2 C:\xampp\htdocs\yii_main\framework\base\CModule.php(103): CWebApplication->getComponent()
#3 C:\xampp\htdocs\questionaire\protected\modules\user\controllers\LoginController.php(12): CWebApplication->__get()
#4 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): LoginController->actionLogin()
#5 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): LoginController->runAction()
#7 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): LoginController->runActionWithFilters()
#8 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): LoginController->run()
#9 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\questionaire\index.php(13): CWebApplication->run()
REQUEST_URI=/questionaire/index.php
in C:\xampp\htdocs\questionaire\protected\modules\user\controllers\LoginController.php (12)
in C:\xampp\htdocs\questionaire\index.php (13)
2013/06/21 00:02:32 [error] [php] include(RWebUser.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii_main\framework\YiiBase.php:421)
Stack trace:
#0 C:\xampp\htdocs\yii_main\framework\YiiBase.php(196): import()
#1 C:\xampp\htdocs\yii_main\framework\base\CModule.php(386): createComponent()
#2 C:\xampp\htdocs\yii_main\framework\base\CModule.php(103): CWebApplication->getComponent()
#3 C:\xampp\htdocs\questionaire\protected\modules\user\controllers\LoginController.php(12): CWebApplication->__get()
#4 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): LoginController->actionLogin()
#5 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): LoginController->runAction()
#7 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): LoginController->runActionWithFilters()
#8 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): LoginController->run()
#9 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\questionaire\index.php(13): CWebApplication->run()
REQUEST_URI=/questionaire/index.php
in C:\xampp\htdocs\questionaire\protected\modules\user\controllers\LoginController.php (12)
in C:\xampp\htdocs\questionaire\index.php (13)
2013/06/21 00:11:41 [error] [php] include(RController.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii_main\framework\YiiBase.php:421)
Stack trace:
#0 C:\xampp\htdocs\questionaire\protected\components\Controller.php(7): spl_autoload_call()
#1 C:\xampp\htdocs\yii_main\framework\YiiBase.php(421): include()
#2 unknown(0): autoload()
#3 C:\xampp\htdocs\questionaire\protected\modules\user\controllers\LoginController.php(4): spl_autoload_call()
#4 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(354): require()
#5 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(338): CWebApplication->createController()
#6 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(276): CWebApplication->createController()
#7 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\xampp\htdocs\questionaire\index.php(13): CWebApplication->run()
REQUEST_URI=/questionaire/index.php
in C:\xampp\htdocs\questionaire\protected\components\Controller.php (7)
in C:\xampp\htdocs\questionaire\protected\modules\user\controllers\LoginController.php (4)
in C:\xampp\htdocs\questionaire\index.php (13)
2013/06/21 00:13:19 [error] [php] include(RWebUser.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii_main\framework\YiiBase.php:421)
Stack trace:
#0 C:\xampp\htdocs\yii_main\framework\YiiBase.php(196): import()
#1 C:\xampp\htdocs\yii_main\framework\base\CModule.php(386): createComponent()
#2 C:\xampp\htdocs\yii_main\framework\base\CModule.php(103): CWebApplication->getComponent()
#3 C:\xampp\htdocs\questionaire\protected\modules\user\controllers\LoginController.php(12): CWebApplication->__get()
#4 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): LoginController->actionLogin()
#5 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): LoginController->runAction()
#7 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): LoginController->runActionWithFilters()
#8 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): LoginController->run()
#9 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\questionaire\index.php(13): CWebApplication->run()
REQUEST_URI=/questionaire/index.php
in C:\xampp\htdocs\questionaire\protected\modules\user\controllers\LoginController.php (12)
in C:\xampp\htdocs\questionaire\index.php (13)
2013/06/21 00:21:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "registration".' in C:\xampp\htdocs\yii_main\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii_main\framework\web\CController.php(270): CController->missingAction('registration')
#1 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): CController->run('registration')
#2 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController('site/registrati...')
#3 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\questionaire\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/questionaire/site/registration
---
2013/06/21 00:21:36 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'questionaire.que_profiles' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `que_profiles`.
in C:\xampp\htdocs\questionaire\protected\modules\user\controllers\RegistrationController.php (24)
in C:\xampp\htdocs\questionaire\index.php (13)
2013/06/21 00:21:36 [error] [exception.CDbException] exception 'CDbException' with message 'The table "{{profiles}}" for active record class "Profile" cannot be found in the database.' in C:\xampp\htdocs\yii_main\framework\db\ar\CActiveRecord.php:2310
Stack trace:
#0 C:\xampp\htdocs\yii_main\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Profile))
#1 C:\xampp\htdocs\yii_main\framework\db\ar\CActiveRecord.php(402): CActiveRecord::model('Profile')
#2 C:\xampp\htdocs\yii_main\framework\db\ar\CActiveRecord.php(77): CActiveRecord->getMetaData()
#3 C:\xampp\htdocs\questionaire\protected\modules\user\controllers\RegistrationController.php(24): CActiveRecord->__construct()
#4 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): RegistrationController->actionRegistration()
#5 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): CController->run('')
#9 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController('user/registrati...')
#10 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\questionaire\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/questionaire/user/registration
---
2013/06/21 00:33:25 [error] [php] include(WebUser.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii_main\framework\YiiBase.php:421)
Stack trace:
#0 C:\xampp\htdocs\yii_main\framework\YiiBase.php(196): import()
#1 C:\xampp\htdocs\yii_main\framework\base\CModule.php(386): createComponent()
#2 C:\xampp\htdocs\yii_main\framework\base\CModule.php(103): CWebApplication->getComponent()
#3 C:\xampp\htdocs\questionaire\protected\modules\user\controllers\LoginController.php(12): CWebApplication->__get()
#4 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): LoginController->actionLogin()
#5 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): LoginController->runAction()
#7 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): LoginController->runActionWithFilters()
#8 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): LoginController->run()
#9 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\questionaire\index.php(13): CWebApplication->run()
REQUEST_URI=/questionaire/user/login
in C:\xampp\htdocs\questionaire\protected\modules\user\controllers\LoginController.php (12)
in C:\xampp\htdocs\questionaire\index.php (13)
2013/06/24 07:05:23 [error] [php] include(LoginForm.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii_main\framework\YiiBase.php:421)
Stack trace:
#0 C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php(81): spl_autoload_call()
#1 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#2 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): SiteController->runAction()
#4 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): SiteController->runActionWithFilters()
#5 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): SiteController->run()
#6 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController()
#7 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\xampp\htdocs\questionaire\index.php(13): CWebApplication->run()
REQUEST_URI=/questionaire/site/login
in C:\xampp\htdocs\questionaire\protected\controllers\SiteController.php (81)
in C:\xampp\htdocs\questionaire\index.php (13)
2013/06/24 07:07:05 [error] [php] Undefined variable: profile (C:\xampp\htdocs\questionaire\protected\modules\user\views\user\registration.php:28)
Stack trace:
#0 C:\xampp\htdocs\yii_main\framework\web\CController.php(869): RegistrationController->renderFile()
#1 C:\xampp\htdocs\yii_main\framework\web\CController.php(782): RegistrationController->renderPartial()
#2 C:\xampp\htdocs\questionaire\protected\modules\user\controllers\RegistrationController.php(70): RegistrationController->render()
#3 C:\xampp\htdocs\yii_main\framework\web\actions\CInlineAction.php(49): RegistrationController->actionRegistration()
#4 C:\xampp\htdocs\yii_main\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii_main\framework\web\CController.php(286): RegistrationController->runAction()
#6 C:\xampp\htdocs\yii_main\framework\web\CController.php(265): RegistrationController->runActionWithFilters()
#7 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(282): RegistrationController->run()
#8 C:\xampp\htdocs\yii_main\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii_main\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\questionaire\index.php(13): CWebApplication->run()
REQUEST_URI=/questionaire/user/registration
in C:\xampp\htdocs\questionaire\protected\modules\user\views\user\registration.php (28)
in C:\xampp\htdocs\questionaire\protected\modules\user\controllers\RegistrationController.php (70)
in C:\xampp\htdocs\questionaire\index.php (13)
2013/06/24 07:09:26 [warning] [application] Failed to set unsafe attribute "verifyCode" of "RegistrationForm".
in C:\xampp\htdocs\questionaire\protected\modules\user\controllers\RegistrationController.php (27)
in C:\xampp\htdocs\questionaire\index.php (13)
2013/06/24 07:09:28 [warning] [application] Failed to set unsafe attribute "verifyCode" of "RegistrationForm".
in C:\xampp\htdocs\questionaire\protected\modules\user\controllers\RegistrationController.php (27)
in C:\xampp\htdocs\questionaire\index.php (13)
